Makefile

```diff
@@ -6,7 +6,7 @@ VERSION :=
 MOD ?= vendor
 DOCS_DIR ?= docs/cli
 
-ALL_BUILD_TAGS := "aws,packet,aks,e2e,disruptivee2e,poste2e"
+ALL_BUILD_TAGS := "aws,packet,aks,e2e,baremetal,disruptivee2e,poste2e"
```

Comment:
This being split with previous commit makes the previous commit (1st commit in the series) just do half of the things (add tags but not compile it when it should). I don't see the point to split it.

Am I missing something?

Reply:
The scope of the change is different. 1st commit introduces the tests. 2nd commit enables them, as it touches different area of code. It allows to disable the test without removing the code and vice versa in the future. For me it seems like a reasonable split.